Makefile

```diff
@@ -13,6 +13,7 @@ default: all
 VERSION ?= us
 # If COMPARE is 1, check the output sha1sum when building 'all'
 COMPARE ?= 1
+export LANG := C
```

Comment:
does the decomp.me decompile button still work after adding this line?
`./import src asm --decompme`

Reply:
I try and yes it still work